.github/workflows/upload.yml

```
@@ -12,7 +12,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.10
+          python-version: "3.10"

```

```suggestion
          python-version: '3.10'
```
Since we want it as a literal string and not escaping any sequences, better to use single quotes